util/imagetools/printers.go

```diff
@@ -33,6 +38,7 @@ type Printer struct {
 	manifest  ocispecs.Descriptor
 	index     ocispecs.Index
 	platforms []ocispecs.Platform
+	pkeys     []string
```

Comment:
Why does this need to be in a struct and not just local variable in `printBuildInfos` ?